runtime/xamarin/main.h

```diff
@@ -63,6 +63,18 @@ enum XamarinGCHandleType : int {
 	XamarinGCHandleTypePinned = 3,
 };
 
+// Keep in sync with Runtime.LookupTypes in Runtime.CoreCLR.cs
+enum XamarinLookupTypes : int {
```

Comment:
The default underlying enum type is undefined in C (“big enough to fit all the values” - and that’s it, if it fits in a byte, it could be a byte, but it might be a long too). So I prefer to be explicit, especially when there’s a matching enum in C#.